src/frontends/onnx/frontend/src/utils/convpool.cpp

```diff
@@ -135,19 +135,30 @@ void calculate_auto_pads(const Shape& data_shape,
                          CoordinateDiff& padding_below,
                          CoordinateDiff& padding_above) {
     if (pad_type == ngraph::op::PadType::SAME_UPPER || pad_type == ngraph::op::PadType::SAME_LOWER) {
-        padding_below.clear();
-        padding_above.clear();
-        // Extract kernel shape - remove (N,C) channels
-        Shape kernel_shape(std::next(std::begin(filter_shape), 2), std::end(filter_shape));
-        OPENVINO_SUPPRESS_DEPRECATED_START
-        ngraph::infer_auto_padding(data_shape,
-                                   kernel_shape,
-                                   strides,
-                                   dilations,
-                                   pad_type,
-                                   padding_above,
-                                   padding_below);
-        OPENVINO_SUPPRESS_DEPRECATED_END
+        // Move `try_apply_auto_padding` padding for static shape implementation to this function
+        // as it is only used here. The `calculate_auto_pads` is not used in the project could be removed ?
+        //
+        // In general the  auto padding is  calculated for openvino operators like convolution, pooling
+        // when created and shape inference performed
```

Comment:
Is the comment required? I see no value for users in it
Better to start discussion about `calculate_auto_pads` deletion offline

Reply:
Not required, removed.